### DISCUSSION TITLE
Issue: Wifi not working properly

### DISCUSSION THREAD

**Discussion by NomadicCore** [AerynOS/discussions#7] [Q&A] [answered]
Hardware: Framework 13 7840U (AMD RZ616 wifi card)
SerpentOS: 0.4 synced to 11th September 2024
Kernel: 6.10.9-43
Desktop: Cosmic (although issue present in Gnome)

**Issue:** 
Wifi does not work on any network other than my home wifi network.

**Testing:**
Home network: Unifi setup with four U6LR access points around the house. The wifi connects and does not give any issues
In-laws network: Unifi setup with four NanoHD access points (wifi 5 based) around the house. The laptop will not connect to the wifi
My parents network: Same as in-laws with four NanoHD access points and will not connect
Apple iphone 14 hotspot: Will not connect
Google Pixel 6 hotspot: Will not connect
Google Pixel 8Pro hotspot: Will not connect

![screenshot-2024-09-11-08-09-04](https://github.com/user-attachments/assets/c34335f5-c663-40a1-b31a-a1faa107919c)
Screenshot from cosmic DE trying to connect to Google Pixel 8 Pro

**Additional context:**
I'm not sure if the wifi was working on an earlier release last month and then something has broken since. Maybe the credentials etc are working fine for an already established network (my home network) but it can't successfully connect to any new network. I don't want to forget my home network if this issue persists and I can't connect to wifi at home.

**Comment by endumiuz** [comment, marked as answer]
I'm also unable to connect to WiFi through GUI (Cosmic), but `sudo nmcli device wifi connect <SSID> -a` works for me.

**Reply by NomadicCore** [reply]
Thanks,

Will try this out
